mmcv/runner/fp16_utils.py

```diff
@@ -52,18 +56,22 @@ def cast_tensor_type(inputs, src_type, dst_type):
     elif isinstance(inputs, np.ndarray):
         return inputs
     elif isinstance(inputs, abc.Mapping):
-        return type(inputs)({
+        return type(inputs)({  # type: ignore
```

Comment:
If we change the type hints of the return value, will this `type: ignore` still be needed?

Reply:
![image](https://user-images.githubusercontent.com/92794867/170829060-300b5441-e174-4b8e-9f7e-257bcc77f697.png)
Maybe not.
It seems like a little difficult to pass